app/assets/javascripts/components/widgets_box.js.jsx

```diff
@@ -68,40 +68,54 @@ var Widget = React.createClass({
   },
 
   render(){
-    var rawMarkup = marked(this.props.children.toString(), {sanitize: true});
```

Comment:
Is this to add markdown support to our body text editor? I'm down, but I'm skeptical that our campaigners are big on markdown.